geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/connection/AuthIntegrationTest.java

```diff
@@ -77,13 +93,15 @@ private void setupCache(boolean withUsername, boolean withSecurityManager) throw
     server = new GeodeRedisServer("localhost", port, (InternalCache) cache);
     server.getRegionProvider().getSlotAdvisor().getBucketSlots();
     this.jedis = new Jedis("localhost", port, 100000);
+
+    System.clearProperty("io.netty.eventLoopThreads");
```

Comment:
Does this need to be in a finally block perhaps?

Reply:
Good idea.